tests/storage_tests.cpp

```diff
@@ -4,6 +4,26 @@
 
 using namespace sqlite_orm;
 
+#ifdef SQLITE_ORM_CTAD_SUPPORTED
+TEST_CASE("connection control") {
+    const auto openForever = GENERATE(false, true);
+    SECTION("") {
```

Comment:
what is the purpose of empty single session?

Reply:
`GENERATE` repeats the `SECTION` following it.